config/initializers/carrierwave.rb

```diff
@@ -14,14 +15,24 @@
     config.storage = :file
   else
     config.fog_provider = "fog/aws"
-    region = ApplicationConfig["AWS_UPLOAD_REGION"].presence || ApplicationConfig["AWS_DEFAULT_REGION"]
-    config.fog_credentials = {
-      provider: "AWS",
-      aws_access_key_id: ApplicationConfig["AWS_ID"],
-      aws_secret_access_key: ApplicationConfig["AWS_SECRET"],
-      region: region
-    }
-    config.fog_directory = ApplicationConfig["AWS_BUCKET_NAME"]
     config.storage = :fog
+    if ENV["HEROKU_APP_ID"].present?
+      region = ApplicationConfig["AWS_UPLOAD_REGION"].presence || ApplicationConfig["AWS_DEFAULT_REGION"]
+      config.fog_credentials = {
+        provider: "AWS",
+        aws_access_key_id: ApplicationConfig["AWS_ID"],
+        aws_secret_access_key: ApplicationConfig["AWS_SECRET"],
+        region: region
+      }
+      config.fog_directory = ApplicationConfig["AWS_BUCKET_NAME"]
+    else #SYSTEMSNOTE Non-Heroku for our WIP container
+      config.fog_credentials = {
+        provider: "AWS",
+        use_iam_profile: true,
+        region: "us-east-2"
```

Comment:
Any reason not to use `region` here as we do above since they are the same?

Reply:
I was a little unclear about what we had available via ENV vars and what we didn't, so I just left this hardcoded. I was going to follow up with this to get more clarity, but I changed it now with this comment being clarity.